src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -179,88 +177,57 @@ public async Task UpdateRenewalReminderDateAsync(Guid id, DateTime renewalRemind
+                cmd.CommandType = CommandType.StoredProcedure;
+                cmd.Parameters.Add("@Id", SqlDbType.UniqueIdentifier).Value = user.Id;
+                cmd.Parameters.Add("@SecurityStamp", SqlDbType.NVarChar).Value = user.SecurityStamp;
+                cmd.Parameters.Add("@Key", SqlDbType.VarChar).Value = user.Key;
+
+                if (string.IsNullOrWhiteSpace(user.PrivateKey))
                 {
-                    // Update user.
-
-                    using (var cmd = new SqlCommand("[dbo].[User_UpdateKeys]", connection, transaction))
-                    {
-                        cmd.CommandType = CommandType.StoredProcedure;
-                        cmd.Parameters.Add("@Id", SqlDbType.UniqueIdentifier).Value = user.Id;
-                        cmd.Parameters.Add("@SecurityStamp", SqlDbType.NVarChar).Value = user.SecurityStamp;
-                        cmd.Parameters.Add("@Key", SqlDbType.VarChar).Value = user.Key;
-
-                        if (string.IsNullOrWhiteSpace(user.PrivateKey))
-                        {
-                            cmd.Parameters.Add("@PrivateKey", SqlDbType.VarChar).Value = DBNull.Value;
-                        }
-                        else
-                        {
-                            cmd.Parameters.Add("@PrivateKey", SqlDbType.VarChar).Value = user.PrivateKey;
-                        }
-
-                        cmd.Parameters.Add("@RevisionDate", SqlDbType.DateTime2).Value = user.RevisionDate;
-                        cmd.Parameters.Add("@AccountRevisionDate", SqlDbType.DateTime2).Value =
-                            user.AccountRevisionDate;
-                        cmd.Parameters.Add("@LastKeyRotationDate", SqlDbType.DateTime2).Value =
-                            user.LastKeyRotationDate;
-                        cmd.ExecuteNonQuery();
-                    }
-
-                    //  Update re-encrypted data.
-
-                    if (ciphers.Any())
-                    {
-                        // ciphers.UpdateEncryptedData(user.Id, connection, transaction);
-                    }
-
-                    if (folders.Any())
-                    {
-                        // folders.UpdateEncryptedData(user.Id, connection, transaction);
-                    }
-
-                    if (sends.Any())
-                    {
-                        // sends.UpdateEncryptedData(user.Id, connection, transaction);
-                    }
-
-                    if (emergencyAccessKeys.Any())
-                    {
-                        // emergencyAccessKeys.UpdateEncryptedData(user.Id, connection, transaction);
-                    }
-
-                    if (resetPasswordKeys.Any())
-                    {
-                        // resetPasswordKeys.UpdateEncryptedData(user.Id, connection, transaction);
-                    }
-
-                    transaction.Commit();
+                    cmd.Parameters.Add("@PrivateKey", SqlDbType.VarChar).Value = DBNull.Value;
```

Comment:
nit: This format might improve readability.

```suggestion
                cmd.Parameters.Add("@PrivateKey", SqlDbType.VarChar).Value = string.IsNullOrWhiteSpace(user.PrivateKey) ? DBNull.Value : user.PrivateKey;
```

Reply:
Good idea, added!